<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KGVSimplePrintingPageSetupBase</class>
 <widget class="QWidget" name="KGVSimplePrintingPageSetupBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>805</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>kgraphviewer-bev.png</normaloff>kgraphviewer-bev.png</iconset>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <layout class="QVBoxLayout">
     <item>
      <layout class="QVBoxLayout">
       <item>
        <layout class="QVBoxLayout">
         <item>
          <layout class="QVBoxLayout">
           <item>
            <layout class="QVBoxLayout">
             <item>
              <layout class="QVBoxLayout">
               <item>
                <widget class="QLabel" name="captionLabel">
                 <property name="buddy">
                  <cstring>headerTitleLineEdit</cstring>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout">
                 <item>
                  <widget class="QPushButton" name="printButton"/>
                 </item>
                 <item>
                  <widget class="QPushButton" name="printPreviewButton"/>
                 </item>
                 <item>
                  <spacer>
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeType">
                    <enum>QSizePolicy::Expanding</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>278</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QLabel" name="textLabel2">
               <property name="text">
                <string>Pa&amp;ge font:</string>
               </property>
               <property name="buddy">
                <cstring>headerTitleLineEdit</cstring>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout">
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLineEdit" name="headerTitleLineEdit"/>
             </item>
             <item>
              <widget class="QPushButton" name="headerTitleFontButton"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout">
           <item>
            <widget class="QGroupBox" name="groupBox1">
             <property name="title">
              <string>Page Size &amp;&amp; Margins</string>
             </property>
             <layout class="QVBoxLayout">
              <item>
               <widget class="QLabel" name="pageSizeAndMarginsLabel">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="changePageSizeAndMarginsButton"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout">
         <item row="0" column="1">
          <widget class="QCheckBox" name="addTableBordersCheckbox">
           <property name="text">
            <string>Add &amp;borders</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="addDateTimeCheckbox">
           <property name="text">
            <string>Add date and time</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2" rowspan="2">
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>91</width>
             <height>21</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="0">
          <widget class="QCheckBox" name="addPageNumbersCheckbox">
           <property name="text">
            <string>Add page numbers</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox4">
       <property name="title">
        <string>Number of pages to fit on</string>
       </property>
       <layout class="QGridLayout">
        <item row="4" column="1">
         <spacer>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1">
         <spacer>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>21</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0" rowspan="2">
         <widget class="QGroupBox" name="groupBox2">
          <property name="title">
           <string>Horizontally</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QSpinBox" name="horizFitNumInput"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="3" column="0" rowspan="2">
         <widget class="QGroupBox" name="groupBox3">
          <property name="title">
           <string>Vertically</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QSpinBox" name="vertFitNumInput"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="1" rowspan="2">
         <widget class="QPushButton" name="maintainAspectButton">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>.</normaloff>.</iconset>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <layout class="QHBoxLayout">
          <item>
           <widget class="QRadioButton" name="naturalSizeRadioButton">
            <property name="text">
             <string>&amp;Natural size</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="fitToOnePageRadioButton">
            <property name="text">
             <string>Fit &amp;to one page</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="fitToSeveralPagesRadioButton">
            <property name="text">
             <string>F&amp;it to several pages</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout">
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel" name="textLabel6">
           <property name="text">
            <string>Related actions:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="KUrlLabel" name="saveSetupLink"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Close</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KUrlLabel</class>
   <extends>QLabel</extends>
   <header>kurllabel.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>printButton</tabstop>
  <tabstop>printPreviewButton</tabstop>
  <tabstop>headerTitleFontButton</tabstop>
  <tabstop>changePageSizeAndMarginsButton</tabstop>
  <tabstop>addPageNumbersCheckbox</tabstop>
  <tabstop>addTableBordersCheckbox</tabstop>
  <tabstop>addDateTimeCheckbox</tabstop>
  <tabstop>naturalSizeRadioButton</tabstop>
  <tabstop>fitToOnePageRadioButton</tabstop>
  <tabstop>fitToSeveralPagesRadioButton</tabstop>
  <tabstop>horizFitNumInput</tabstop>
  <tabstop>vertFitNumInput</tabstop>
  <tabstop>maintainAspectButton</tabstop>
  <tabstop>saveSetupLink</tabstop>
  <tabstop>headerTitleLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
